aws/build-and-push-to-ecr/main.ts

```diff
@@ -0,0 +1,16 @@
+import * as core from "@actions/core";
+import * as path from "path";
+import execAsync from "../../helpers/execAsync";
+import validateRequiredInputs from "../../helpers/validateRequiredInputs";
```

Comment:
Pourrais tu enlever les default export svp?
```suggestion
import { execAsync } from "../../helpers/execAsync";
import { validateRequiredInputs } from "../../helpers/validateRequiredInputs";
```
Comme cela, si l'on renomme le nom de la fonction, le changement est effectué partout.